gpii/node_modules/gpii-oauth2/gpii-oauth2-authz-server/src/AuthorizationService.js

```diff
@@ -22,6 +22,17 @@ var fluid = fluid || require("infusion");
 
     var gpii = fluid.registerNamespace("gpii");
 
+    fluid.registerNamespace("gpii.oauth2.authorizationService");
+
+    /*
+
```

Comment:
Anomalous spacing here

Reply:
Anomalous in what way?  There are no other comments of that type in the same file.

Reply:
Do the blank lines before and after the line represent an attempt to defeat JSDoc parsing or are they just stray?

Reply:
It's just the way I tend do things for non JS Docs, but I know my style tends to eat up vertical space.  This approach does seem a bit ridiculous for such a short comment, I'll convert it to an inline comment with no extra space.